data/src/main/java/com/antosik/benchproject/data/movies/common/database/MovieDatabase.kt

```diff
@@ -0,0 +1,15 @@
+package com.antosik.benchproject.data.movies.common.database
+
+import androidx.room.Database
+import androidx.room.RoomDatabase
+import com.antosik.benchproject.data.popular.movies.entity.MovieEntity
+
+@Database(
+    entities = [MovieEntity::class],
+    version = 2,
```

Comment:
Why version "2" if you are just adding it 😄 ?

Reply:
I will back to version number 1 :D